05_syntax_lab/01.py

```diff
@@ -1,5 +1,16 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 1
+
+"""

```

Comment:
Hi Dovron,
Love the use of string.format and the early submission :thumbsup:

Starting with big=0 is not the best choice here because users expect the biggest number to always be from their typed in values (and if all values are negative it won't work). Better to read the first value before the loop and read 9 more inside.